WalletWasabi/WabiSabi/Backend/Banning/Inmate.cs

```diff
@@ -23,13 +23,18 @@ public static Inmate FromString(string str)
 		var utxoHashString = parts[2];
 		var utxoIndexString = parts[3];
 		var disruptedRoundIdString = parts[4];
+		var isLongBan = false;
+		if (parts.Length > 5)
+		{
+			isLongBan = bool.Parse(parts[5]);
+		}
 
 		var started = DateTimeOffset.FromUnixTimeSeconds(long.Parse(startedString));
 		var punishment = Enum.Parse<Punishment>(punishmentString);
 		var utxo = new OutPoint(new uint256(utxoHashString), int.Parse(utxoIndexString));
 		var lastDisruptedRoundId = uint256.Parse(disruptedRoundIdString);
 
-		return new(utxo, punishment, started, lastDisruptedRoundId);
+		return new(utxo, punishment, started, lastDisruptedRoundId, isLongBan);
 	}
 
 	public override string ToString()
```

Comment:
`ToString()` method modification is missing.

Reply:
true, fixed